template.yaml

```diff
@@ -77,11 +77,12 @@ Resources:
     Properties:
       FunctionName: !Sub 'CumulusAggFetchAuthorizer-${DeployStage}'
       Handler: src/handlers/site_upload/api_gateway_authorizer.lambda_handler
+      Layers: [arn:aws:lambda:us-east-1:336392948345:layer:AWSSDKPandas-Python311:17]
```

Comment:
Based on the error messages, I :think: this will fix the raised error, though I'm not sure why this is looking for pandas at all. But it works in the dev aggregator, so I'll have to verify this in the deployment, and may need to migrate changes back here..